notify/notify.go

```diff
@@ -206,13 +211,20 @@ func ResolvedAlerts(ctx context.Context) ([]uint64, bool) {
 	return v, ok
 }
 
-// MuteTimeIntervalNames extracts a slice of mute time names from the context. Iff none exists, the
+// MuteTimeIntervalNames extracts a slice of mute time names from the context. If none exists, the
```

Comment:
I bet that Iff was for "if and only if"
```suggestion
// MuteTimeIntervalNames extracts a slice of mute time names from the context. If and only if none exists, the
```